hotsos/client.py

```diff
@@ -380,6 +387,19 @@ def setup_plugin_env(self, plugin):
         HotSOSConfig.plugin_tmp_dir = tempfile.mkdtemp(prefix=plugin,
                                                        dir=global_tmp)
 
+    def clear_plugin_env(self):
+        log.debug("clearing plugin env")
+        HotSOSConfig.plugin_tmp_dir = None
```

Comment:
this should go into teardown_global_env()

Reply:
Done